modules/data/webadmin/tmpl/add_edit_user.tmpl

```diff
@@ -238,9 +238,15 @@
 					<br /><span class="info">Empty = use standard value</span>
 				</div>
 				<div class="subsection third">
-					<div class="inputlabel">Buffer Size:</div>
-					<input type="number" name="bufsize" value="<? VAR BufferCount ?>" min="0"
-						   title="This is the amount of lines that the playback buffer will store before dropping off the oldest line. The buffers are stored in the memory by default." />
+					<div class="inputlabel">Channel Buffer Size:</div>
+					<input type="number" name="chanbufsize" value="<? VAR ChanBufferSize ?>" min="0"
+						   title="This is the amount of lines that the playback buffer will store for channels before dropping off the oldest line. The buffers are stored in the memory by default." />
+					<br /><span class="info">Empty = use standard value</span>
+				</div>
+				<div class="subsection third">
```

Comment:
How about "Channels" and "Queries" sections?

Channels
- Default Modes
- Buffer Size

Queries
- Max Buffers
- Buffer Size